lib/rules/prefer-stateless-function/fixer.js

```diff
@@ -90,7 +86,7 @@ function ruleFixer(sourceCode, componentNode, utils) {
    * @returns {string} stateless component body
    */
   function getComponentText(name, transformedBody) {
-    return `let ${name} = (props) => ${transformedBody};`;
+    return `function ${name}(props) ${transformedBody};`;
```

Comment:
This probably shouldn't include the semicolon either, unless the original text did